template.php

```diff
@@ -479,7 +519,7 @@ function megatron_megatron_btn_dropdown($variables) {
 
   // Finish markup
   $output .= '
-  <div class="ubc7-arrow down-arrow"></div>
+  <div class="ubc7-arrow right-arrow"></div>
```

Comment:
These may break BC, could we use maybe a different theme?

Reply:
Yes, it should be down-arrow. Thank you for bringing this up.